src/software/jetson_nano/services/motor.cpp

```diff
@@ -119,13 +130,38 @@ MotorService::MotorService(const RobotConstants_t& robot_constants,
 MotorService::~MotorService() {}
 
 std::unique_ptr<TbotsProto::DriveUnitStatus> MotorService::poll(
-    const TbotsProto::DirectControlPrimitive_DirectVelocityControl& local_velocity,
-    float dribbler_speed_rpm)
+    const TbotsProto::DirectControlPrimitive& direct_control)
 {
-    // TODO (#2335) convert local velocity to per-wheel velocity
-    // using http://robocup.mi.fu-berlin.de/buch/omnidrive.pdf and then
-    // communicate velocities to trinamic. Also read back feedback and
-    // return drive unit status.
+    CHECK(encoder_calibrated_[FRONT_LEFT_MOTOR_CHIP_SELECT])
+        << "Running without encoder calibration can cause serious harm";
```

Comment:
why only the FRONT_LEFT_MOTOR_CHIP_SELECT?

Reply:
I mentioned in the PR comment but I should probably add it to the code here too, this is still part 2/3

BOBOB can only spin 1 motor rn, added a comment in the code too